<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="panel">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-10</y>
      <width>671</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>water.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bienvenida">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BIENVENIDOS A TIC TAC TOE</string>
    </property>
   </widget>
   <widget class="QLabel" name="JvsJ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>JUGADOR VS JUGADOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jvsjButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>EMPEZAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="jvscpu">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>JUGADOR VS CPU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jcpu">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>EMPEZAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="img2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>o.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="img3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>x.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="img1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>o.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="img3_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>x.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bienvenidaImg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>bienvenida.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="finalizar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FINALIZAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
